pkg/openstack/loadbalancer_test.go

```diff
@@ -1108,3 +1108,130 @@ func Test_getStringFromServiceAnnotation(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildBatchUpdateMemberOpts(t *testing.T) {
+
+	// Sample Nodes
+	node1 := &corev1.Node{
+		ObjectMeta: v1.ObjectMeta{
+			Name: "node-1",
+		},
+		Status: corev1.NodeStatus{
+			Addresses: []corev1.NodeAddress{
+				{
+					Type:    corev1.NodeInternalIP,
+					Address: "192.168.1.1",
+				},
+			},
+		},
+	}
+	node2 := &corev1.Node{
+		ObjectMeta: v1.ObjectMeta{
+			Name: "node-2",
+		},
+		Status: corev1.NodeStatus{
+			Addresses: []corev1.NodeAddress{
+				{
+					Type:    corev1.NodeInternalIP,
+					Address: "192.168.1.2",
+				},
+			},
+		},
+	}
+
+	// Instantiate an object of your LbaasV2 struct
+	lbaas := &LbaasV2{}
```

Comment:
Move this declaration in the `t.Run` loop, so that use use a fresh `LbaasV2` object each time